backend/api_volume_test.go

```diff
@@ -0,0 +1,51 @@
+package backend
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+var volumeToNGNT = "/volume?code=NGNT&issuer=GAWODAROMJ33V5YDFY3NPYTHVYQG7MJXVJ2ND3AOGIHYRWINES6ACCPD"
+var volumeFromCENTUS = "/volume?code=CENTUS&issuer=GAKMVPHBET4T7DPN32ODVSI4AA3YEZX2GHGNNSBGFNRQ6QEVKFO4MNDZ&volumeFrom=true"
+
+func TestVolumeHandler(t *testing.T) {
+	tests := []queryTest{
+		{
+			name:           "volumeTo full history",
+			r:              httptest.NewRequest("GET", volumeToNGNT, nil),
+			w:              httptest.NewRecorder(),
+			expectedStatus: http.StatusOK,
+			golden:         "NGNT_all.golden",
```

Comment:
Smart use of golden files here to avoid networked requests! Good job applying learnings from the ETL project to this one :)